src/main/java/com/yunus1903/chatembeds/client/ChatScrollPos.java

```diff
@@ -0,0 +1,13 @@
+package com.yunus1903.chatembeds.client;
+
+public class ChatScrollPos {
```

Comment:
This class will probably need to go.

I don't know how to access a `private` field in a Minecraft class, so this is used to get around `chatScrollbarPos` being private in `ChatComponent`

Reply:
You can transform `private` fields to be more permissive using [Access Transformers](https://mcforge.readthedocs.io/en/latest/advanced/accesstransformers/).